QMLFile.js

```diff
@@ -52,8 +52,10 @@ QMLFile.unescapeString = function(string) {
         replace(/^\\\\/, "\\").             // unescape backslashes
         replace(/([^\\])\\\\/g, "$1\\").
         replace(/^\\"/, '"').
+        replace(/^\\'/, "'").
         replace(/([^\\])\\"/g, '$1"').
-        replace(/\\"/g, '"');
+        replace(/\\"/g, '"').
+        replace(/\\'/g, "'");
```

Comment:
This maps \\' to ' twice, no?

Reply:
The problem case was `qsTr("Don\'t save")`, loctool converted it as `"Don\\'t save"` so couldn't find the translation properly. 
I don't think it maps \' twice. am I missing some other case?

Reply:
Line 55 and 58 both map the escaped single quote, though line 55 has the "beginning of line" carat in it for some reason. Line 58 should catch both cases

Reply:
ah..I update it.